packages/trpc/react/trpc.ts

```diff
@@ -43,22 +43,27 @@ export type Endpoint = (typeof ENDPOINTS)[number];
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const resolveEndpoint = (links: any) => {
-  // TODO: Update our trpc routes so they are more clear.
   // This function parses paths like the following and maps them
   // to the correct API endpoints.
   // - viewer.me - 2 segment paths like this are for logged in requests
   // - viewer.public.i18n - 3 segments paths can be public or authed
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   return (ctx: any) => {
+    // split the segmented path into an array
```

Comment:
I don't think this comment is needed. `split` makes it obvious that we are converting into an array.